### PR TITLE
Add lookup tables as git submodules

### PR DESCRIPTION
**Pull request by dr0i**
Do [edited:] `git config --local submodule.recurse true` once in your checked out repo. Any further `git pull` will automatically update the submodules.

See  #1893.

### CODE REVIEW

**Review by blackwinter — APPROVED** [review]
> Do "git config --global submodule.recurse true" once in your checked out repo.

I don't think that's such a good recommendation. I'd suggest `--local` instead (which is the default). Or just use `git pull --recurse-submodules` explicitly.

**Comment by dr0i** [comment]
`--local` is good. (I like to use the "established" commands  (here `git pull`) to work with the new setting - not only because of cronjobs and scripts (which will work without adapting them).

**Comment by TobiasNx** [comment]
` src/test/resources/deweyLabels.tsv` should also be moved into the maps folder.

**Comment by TobiasNx** [comment]
Also we do not need a specific `src/test/resources/alma/maps` folder a folder `src/test/resources/maps` would be good enough.

**Comment by dr0i** [comment]
We have `src/main/resources/alma/maps/` and thus `src/test/resources/alma/maps/` is symmetric - which is a very nice property and should be kept.

Re dewey: done that
Re further discussion of where to put test resources: that's not the scope of this PR/issue so  we should open a new one if needed.

**Comment by dr0i** [comment]
I've deleted now `src/main/resources/alma/maps/generatedAlmaSuppressedLocations.tsv`.
Re documentation: I don't understand why we should put a readme into the `maps` directory when these lookup tables are not read from this directory.

**Comment by TobiasNx** [comment]
> I've deleted now `src/main/resources/alma/maps/generatedAlmaSuppressedLocations.tsv`. Re documentation: I don't understand why we should put a readme into the `maps` directory when these lookup tables are not read from this directory.

okay, then go for it.
I just thought that it helps to find missing map files when somebody does not find it in the folder. But then go on. +1

**Review by TobiasNx — APPROVED** [review]

File: README.textile [hunk 1]
```diff
@@ -102,6 +102,10 @@ If you've added and commited everything, check again if all is ok:
 
 This should result in *BUILD SUCCESS*. Push your changes. You're done :)
 
+h2. Tables as gitsubmodules
+The lookup tables are provided through gitsubmodules. To make a @git pull@ also
```
Comment (TobiasNx) [inline README.textile:106]: Oh, perhaps say "Some lookup tables" since not all of them are provided via submodule.

Comment (dr0i) [reply README.textile]: Resoved with [8338ca9](https://github.com/hbz/lobid-resources/pull/1902/commits/8338ca93ec74af0db9c9ebc687ff32aea5b04856).

**Comment by dr0i** [comment]
Unfortunatley it seem not possible to have the submodules not in a detached state (may depend on the git version) but you have to do ` git submodule update --recursive --remote` after `git pull`. Will update all scripts and README.

**Pull request merged**